src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -872,8 +872,8 @@ test_run_mb(void)
 
 	test_captbl_expand();
 
-	test_wakeup();
-	test_preemption();
+//	test_wakeup();
```

Comment:
Do you really want to not run these tests?  Are they now broken and need fixing?  What's happening here?

Reply:
I've commented the preemption-stack logic in the kernel, so I commented the test cases related to it. But I think `test_wakeup()` still runs fine and not `test_preemption()`.

Reply:
I've fixed it. It's a GitHub thing that we don't see the newer commits inline to your previous comments.